vendor/github.com/weaveworks/common/middleware/logging.go

```diff
@@ -91,9 +83,6 @@ func (l Log) Wrap(next http.Handler) http.Handler {
 		statusCode, writeErr := wrapped.getStatusCode(), wrapped.getWriteError()
 
 		if writeErr != nil {
-			if errors.Is(writeErr, DoNotLogError{}) {
```

Comment:
Isn't required anymore? Can you explain why was it removed from here?

Reply:
This error is the return from `Write()`; I don't see how it can be one of our wrapped errors.

Reply:
Also I looked at the high-volume log lines that have been problemmatic and they were all from grpc_logging, none from http request logging.